### PR TITLE
AF-2644 : Package list fails to populate default package for imported projects

### PR DESCRIPTION
**Pull request by Rikkola**
**JIRA**: [AF-2644](https://issues.redhat.com/browse/AF-2644)

Problem is that there are two empty packages. One shows up as "nothing selected" and the other as "<default>". Removing one of them.

### CODE REVIEW

**Comment by jstastny-cz** [comment]
jenkins do fdb

File: kie-wb-common-services/kie-wb-common-services-backend/src/main/java/org/kie/workbench/common/services/backend/project/KieModuleServiceImpl.java [hunk 1]
```diff
@@ -135,15 +135,17 @@ public KieModulePackages resolveModulePackages(Module activeModule) {
                                      PackageItem.DEFAULT_PACKAGE_NAME));
 
         for (final String packageName : packageServiceLoader.find(activeModule.getRootPath())) {
-            final StringBuilder packageBuilder = new StringBuilder();
-            for (final String token : packageName.split("\\.")) {
-                if (packageBuilder.length() != 0) {
-                    packageBuilder.append(".");
+            if (!packageName.trim().isEmpty()) {
```
Comment (Rikkola) [inline kie-wb-common-services/kie-wb-common-services-backend/src/main/java/org/kie/workbench/common/services/backend/project/KieModuleServiceImpl.java:138]: The change is this if-line.

**Comment by jstastny-cz** [comment]
jenkins do fdb

**Comment by jstastny-cz** [comment]
Summary from our discussion:
* populated value is `<default>` for imported projects (using Try Samples project import)
* we will incorporate the fix within this PR

**Comment by Rikkola** [comment]
@jstastny-cz Ok so looks like the problem is just with the DMN example. This is because the default package is formed from GAV and in this project it differs with the existing packages. I guess for this reason it is hard to know what the default package would be and fix might not be needed.

**Comment by jstastny-cz** [comment]
Confirming that for imported projects the behavior is following:
* when package and GAV are in sync then the value in dialog is the expected package
* when package and GAV are not in sync then the value in dialog is <default>

**Review by jstastny-cz — APPROVED** [review]

**Review by vpellegrino — APPROVED** [review]
LGTM, thanks @Rikkola

**Comment by Rikkola** [comment]
Jenkins run fdb

**Comment by Rikkola** [comment]
@kiegroup/gatekeepers Can I get a merge please.

**Pull request merged**